esmvalcore/cmor/_fixes/cmip5/bcc_csm1_1_m.py

```diff
@@ -1,12 +1,16 @@
-
 """Fixes for bcc-csm1-1-m."""
 import numpy as np
 from scipy.interpolate import InterpolatedUnivariateSpline
 from scipy.ndimage import map_coordinates
 
+from .bcc_csm1_1 import Cl as BaseCl
 from ..fix import Fix
 
 
+class Cl(BaseCl):
+    """Fixes for cl."""
+
+
```

Comment:
```suggestion
from ..fix import Fix

from .bcc_csm1_1 import Cl
```
Should have the exact same effect. No need to derive a new class, I think.

Iff true (check with one model first), this applies of course to all similar files.

Reply:
:+1: